man/froll.Rd

```diff
@@ -104,9 +106,13 @@
   \itemize{
     \item \code{align} does not support \code{"center"}.
     \item if list of vectors is passed to \code{x}, then all vectors within it must have equal length.
+    \item \code{partial=TRUE} is not supported.
     \item functionality is not supported in \code{frollapply} (to be changed).
   }
 }
+\section{\code{partial} argument}{
+  \code{partial=TRUE} will turn a function into adaptive function and trim window size in \code{n} argument using \code{n = c(seq.int(n), rep(n, len-n))} to available observations. It inherits limitations of adaptive rolling functions, see above. Adaptive functions uses more complex algorithms, therefore if performance is important then \code{partial=TRUE} should be avoided in favour of computing missing observations "by hand" after the rolling function; see examples.
```

Comment:
Something like this

```suggestion
  \code{partial=TRUE} is a convenient way to express \code{adaptive=TRUE} for the common case of trim window size in \code{n} being \code{n = c(seq.int(n), rep(n, len-n))} to available observations. It inherits limitations of adaptive rolling functions, see above. Adaptive functions use more complex algorithms; if performance is important, \code{partial=TRUE} should be avoided in favour of computing missing observations "by hand" after the rolling function; see examples.
```

Reply:
This does make me wonder -- why implement it the slow way, then? What's keeping us from doing this ex-post touch-up on behalf of the user for optimal performance?

Reply:
I would say extra work needed, extra complexity added to the code, and small users demand.
Ideally I would say is to create an issue where users can upvote, and we see how important it is.